Lib/sysconfig.py

```diff
@@ -204,20 +224,13 @@ def joinuser(*args):
     return joinuser("~", ".local")
 
 
-def _parse_makefile(filename, vars=None):
+def _parse_makefile(filename, vars=None, distutils_compat=False):
```

Comment:
Can we rename `distutils_compat` to something like `keep_unresolved` (or a name that makes more sense... I've only quickly read over to see what it's actually doing)

Reply:
Good idea and right timing before I change docs. I'm gonna implement it.